### ISSUE TITLE
Language Contact and Relatedness in the Hindukush Region

### ISSUE THREAD

**Issue by tresoldi** [lexibank/lexibank#168]
Project led by Henrik Liljegren - https://hindukush.ling.su.se/

**Comment by LinguList** [comment]
This dataset is very interestin, as it also has grammatical features, so a case for cldfbench + lexibank, right, @xrotwang ?

Features can be downloaded already, wordlists not yet.

**Comment by tresoldi** [comment]
It looks more like a bug, as the vocabulary page has a download button and he mentioned something about collaboration and open data when presenting in Leipzig.

**Comment by LinguList** [comment]
Oh, you know him? Could you contact him and ask him if he could share the data on zenodo?

**Comment by tresoldi** [comment]
No, I saw his presentation at SLE and had a 5-second "nice talk" interaction. But I will take of contacting him, adding to my TODO list.

**Comment by xrotwang** [comment]
I know him, he also has a dictionary on [dictionaria](https://dictionaria.clld.org/contributions/palula).
